http-clients/apache-client/src/main/java/software/amazon/awssdk/http/apache/ApacheHttpClient.java

```diff
@@ -304,6 +311,27 @@ default Builder apply(Consumer<Builder> mutator) {
          * Configure the maximum amount of time that a connection should be allowed to remain open while idle.
          */
         Builder connectionMaxIdleTime(Duration maxIdleConnectionTimeout);
+
+        /**
+         * Forces the client to trust all certificates, even invalid or self signed certificates. This should only ever
+         * be used for testing purposes.
+         *
+         * @param trustAllCertificates Whether to trust all certificates. The default is false and only valid certificates
+         *                             whose trust can be verified via the trust store will be trusted.
+         * @return This builder for method chaining.
+         */
+        Builder trustAllCertificates(Boolean trustAllCertificates);
```

Comment:
We should probably just use the primitive `boolean` type

Reply:
was trying to be consistent with other boolean option in the class "expectContinueEnabled".